config/application.rb

```diff
@@ -125,8 +125,8 @@ def self.load_version
     config.thumbnail_width = 75
 
     # UMM Versions
-    config.umm_c_version = '1.10'
-    config.umm_v_version = '1.1'
-    config.umm_s_version = '1.1'
+    config.umm_c_version   = '1.10'
```

Comment:
Just had to have it all line up, huh?

Reply:
haha yes. It's... 'more readable' 😬 #ocd